develop/schemadoc/Protocol/Protocol.HTTP.Session.Connection.Request.Parameters.Parameter.md

```diff
@@ -27,6 +27,9 @@ Via HTTP, this has to be done by means of key/value pairs.
 
 Contains a string that specifies the value of the parameter.
 
+> [!NOTE]
+> When you provide the value to be sent through the PID of a parameter, that parameter needs to be of RawType `other` and Type `string`.
```

Comment:
Only mention Type "string" requirement.
This note could also be added to the second note in this section: https://docs.dataminer.services/develop/devguide/Connector/ConnectionsHttpImplementing.html#request
(I checked the code and this does not seem to apply for Data)